components/ImageCard/ImageCard.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ReactElement } from "react"
+import styles from "@/styles/components/ImageCard.module.scss"
+
+interface ImageCardProps {
+  imageSrc: string
+  children: ReactElement
+}
+
+const ImageCard = ({ imageSrc, children }: ImageCardProps) => {
+  return (
+    <article className={styles.imageCardContainer}>
```

Comment:
Why the use of `article`? We should get Clare's input on this.

Reply:
I've seen article used as the semantic tag for this type of element in the past, but it could just as easily be a div or something else. Happy to consult Clare on this.

Reply:
This was a question out of curiosity and not mean to be something that you _had_ to update.